### PR TITLE
SophosLimited.SophosSetup version 1.7.6.0

### PR DESCRIPTION
**Pull request by VikramKondabathini**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50162)

### CODE REVIEW

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
+InstallerSwitches:
+  Silent: h
+  SilentWithProgress: Silent
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml:8]:
```suggestion
  Silent: --quiet
  SilentWithProgress: --quiet
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
+PackageLocale: en-US
+Publisher: Sophos Limited
+PackageName: Sophos
+License: Copyright 1989-2020 Sophos Limited.
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
Copyright: Copyright 1989-2020 Sophos Limited.
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
+PackageLocale: en-US
+Publisher: Sophos Limited
+PackageName: Sophos
+License: Copyright 1989-2020 Sophos Limited.
+ShortDescription: GznSophos
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml:10]: Please provide a better short description

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: SophosLimited.SophosEndpointAgent
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml:4]:
```suggestion
PackageIdentifier: SophosLimited.SophosEndpointAgent
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.yaml:4]:
```suggestion
PackageIdentifier: SophosLimited.SophosEndpointAgent
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.20.11
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml:5]:
```suggestion
PackageVersion: 2.20.11
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.yaml:5]:
```suggestion
PackageVersion: 2.20.11
```

File: manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
+InstallerSwitches:
+  Silent: h
+  SilentWithProgress: Silent
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://shareext.blob.core.windows.net/mis/SophosSetup.exe
```
Comment (Trenly) [inline manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml:12]: This doesn't look like an official Sophos URL
Also, the installer doesn't seem to exit after a successful install

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by VikramKondabathini** [comment]
have to change some thing

**Pull request closed without merging**